docs/input-manifest/api-reference.md

```
@@ -227,6 +228,32 @@ To find out how to configure Azure provider and service account, follow the inst
   - `tag`: Optional. If set when the git repository is downloaded, the commit hash from the tag version is used.
   - `path`: specifies the path for a specific provider within the `repository` where the source template files are located.
 
+### OVHcloud
+
+The fields that need to be included in a Kubernetes Secret resource to utilize the OVHcloud provider.
+To find out how to configure OVHcloud provider and API credentials, follow the instructions [here](./providers/ovh.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)